testing/backup-manager/APKBUILD

```diff
@@ -0,0 +1,28 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname="backup-manager"
+pkgver="0.7.15"
+_pkgreal=Backup-Manager
+pkgrel=0
+pkgdesc="Backup Manager is a command line backup tool for GNU/Linux"
+url="https://github.com/sukria/Backup-Manager"
+arch="noarch"
+license="GPL-2.0-only"
+depends="gettext perl bash"
+makedepends="perl-file-slurp"
+options="!check" # tests not works due to dar archiver missing
+source="$pkgname-$pkgver.tar.gz::https://github.com/sukria/Backup-Manager/archive/$pkgver.tar.gz"
+builddir="$srcdir/$_pkgreal-$pkgver"
```

Comment:
Just write Backup-Manager here, making a variable only to use it once afterwards doesn't really make sense.

Reply:
done, thank you.